src/main/java/org/andersen/homework/model/dto/ticket/concert/ConcertTicketUpdateDto.java

```diff
@@ -0,0 +1,20 @@
+package org.andersen.homework.model.dto.ticket.concert;
+
+import java.time.LocalDateTime;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.andersen.homework.model.dto.ticket.TicketUpdateDto;
+import org.andersen.homework.model.enums.StadiumSector;
+
+@Getter
+@Setter
+@ToString(callSuper = true)
+public class ConcertTicketUpdateDto extends TicketUpdateDto {
```

Comment:
What is the difference with ConcertTicketSaveDto?

Reply:
No differences for now, but in future may become required to make some fields unupdatable. It will be really easy to realize with already integrated `ConcertTicketUpdateDto` and `ConcertTicketSaveDto`

Reply:
And what if in the future we will not have such requirements?
From my perspective this is an example of violation YAGNI, DRY and KISS...

If we don't need it currently, I suggest to keep it simpler as could be, for now we don't need 2 different dots separated by ticketType, lets have 1 common